src/DoctrineWriter.php

```diff
@@ -2,11 +2,11 @@
 
 namespace Port\Doctrine;
 
+use Port\Doctrine\Exception\UnsupportedDatabaseTypeException;
```

Comment:
This isn't needed. You can just use `Exception\UnsupportedDatabaseTypeException`.